File: app/src/main/java/com/example/ordinacija/Data/TerminiResultVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class TerminiResultVM {
    public static class Row implements Serializable {
        public int TerminID;
        public Date Datum;
        public String Vrijeme;
        public String Usluga;
        public String Napomena;
        public boolean IsOdobren;
        public boolean IsEvidentiran;
        public boolean IsDeleted;
        public int PacijentID;
        public String ImePrezime;
    }
    public List<TerminiResultVM.Row> rows;
}

File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/TerminiResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class TerminiResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int TerminID { get; internal set; }
            public DateTime Datum { get; set; }
            public string Vrijeme { get; set; }
            public string Napomena { get; set; }
            public bool? IsOdobren { get; set; }
            public bool IsEvidentiran { get; set; }
            public bool IsDeleted { get; set; }

            public string ImePrezime { get; set; }
            public int PacijentID { get; set; }
            public string Usluga { get; set; }


        }
    }
}